src/lib.rs

```diff
@@ -24,3 +25,89 @@ pub fn tags<'a>(
         Ok((key, val))
     })
 }
+
+/// Helper function to iterate through tags from osmflat.
+#[inline]
+pub fn tags_raw<'a>(
```

Comment:
Remove all the non-raw versions (also from flatdata)?

Reply:
Done.